src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -278,15 +278,7 @@ def __read_kv_from_config_store(cmd,
             if keyvault_client and kv.content_type == KeyVaultConstants.KEYVAULT_CONTENT_TYPE:
```

Comment:
>kv.content_type [](start = 35, length = 15)

Can we add a null check for content_type? (Just in case someone decides to specify --fields argument with --resolve-keyvault)

Reply:
Fixed.